RecoLocalCalo/HcalRecAlgos/plugins/HcalRecAlgoESProducer.cc

```diff
@@ -94,5 +96,19 @@ HcalRecAlgoESProducer::produce(const HcalSeverityLevelComputerRcd& iRecord)
    return myComputer ;
 }
 
+void HcalRecAlgoESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    desc.add<uint32_t>("phase",0);
+    desc.add<std::vector<std::string>>("RecoveredRecHitBits");
+    desc.add<std::vector<std::string>>("DropChannelStatusBits");
+    descriptions.add("hcalRecAlgos",desc);
+
+    edm::ParameterSetDescription desc_sevlvl;
+    desc_sevlvl.add<uint32_t>("Level");
+    desc_sevlvl.add<std::vector<std::string>>("RecHitFlags");
+    desc_sevlvl.add<std::vector<std::string>>("ChannelStatus");
+    desc_sevlvl.addVPSet("SeverityLevels",desc_sevlvl);
+}
```

Comment:
@Martin-Grunewald 

Thanks a lot for the detailed instruction. I removed `temp` from the result of `edmPythonConfigToCppValidation`, and got 

```
 void HcalRecAlgoESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
    // hcalRecAlgos
    edm::ParameterSetDescription desc;
    desc.add<unsigned int>("phase", 0);
    desc.add<std::vector<std::string>>("RecoveredRecHitBits", {
            "TimingAddedBit",
            "TimingSubtractedBit",
            });
    {
        edm::ParameterSetDescription vpsd1;
        vpsd1.add<std::vector<std::string>>("RecHitFlags", {
                "",
                });
        vpsd1.add<std::vector<std::string>>("ChannelStatus", {
                "",
                });
        vpsd1.add<int>("Level", 0);
        desc.addVPSet("SeverityLevels", vpsd1);
    }
    desc.add<std::vector<std::string>>("DropChannelStatusBits", {
            "HcalCellMask",
            "HcalCellOff",
            "HcalCellDead",
            });
    descriptions.add("hcalRecAlgos", desc);
}
```

I checked by `runTheMatrix.py -l 10024.0 --useInput all &` that the error is not there anymore.  

What else needs to be done?